cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,23 @@
+import time
+from django.db import connections
+from django.db.utils import OperationalError
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        self.stdout.write("Waiting for database...")
+        db_connection = None
+        while not db_connection:
+            try:
+                db_connection = connections["default"]
+                db_connection.ensure_connection()
+                self.stdout.write("Connecting to database.")
+            except OperationalError:
+                self.stdout.write("Database unavailable.")
```

Comment:
Consider adding more context to this message, such as retrying in a few seconds, to inform the user about the next steps.